uml/backend-software-system/input-listener.puml

```diff
@@ -0,0 +1,53 @@
+@startuml input-listener-console
+package "Console" {
+
+    interface IEventListener {
+        +handler: IEventHandler
+        +native_listen(): List<String>
```

Comment:
just listen is okay, since the bridge has that method name as well.

Reply:
The name of the method will change depending on how it is needed in the bridge with the JNI